### ISSUE TITLE
Installation from cd hangs up without identifying network resources

### ISSUE THREAD

**Issue by Drootsam** [rockstor/rockstor-core#779]
I have been attempting to install Rockstor from a DVD. On the splash screen it allows me to enter time zone, but my network is not identified. I cannot continue. Have I grossly missed something??

**Comment by phillxnet** [comment]
@Drootsam all I can think is that you either don't have DHCP on your network or more likely the network adapter in your system is either not plugged in (or disabled in bios) or not identified by linux. 

Are you installing using an ethernet connection to a know good DHCP network. Rockstor doesn't currently support wifi as far as I know. http://rockstor.com/docs/quickstart.html gives the minimum system requirements.

Note the default install assumes a working DHCP ethernet network connection.
Please give details of your hardware setup regarding the network components so we can see if this is a bug or incompatible hardware.

Thanks

**Comment by Drootsam** [comment]
Yes I am running a wired configuration, not wireless. My Modem Router is a Arris/Motorola SBG6580 "DHCP"
I am using a NAS with 3-3TB Toshiba hard drives, an AMD Phenom 945 x4 on a Micro ASUS  M4N68T-M V2 mother board  with 8GB, and Realtek 8211-VB Lan. I have successfully run this machine on NAS4free, Freenas. I want to try your system before leaving the NAS solution out.


     On Saturday, August 8, 2015 3:22 AM, Philip Guyton <notifications@github.com> wrote:
   

 @Drootsam all I can think is that you either don't have DHCP on your network or more likely the network adapter in your system is either not plugged in (or disabled in bios) or not identified by linux. Are you installing using an ethernet connection to a know good DHCP network. Rockstor doesn't currently support wifi as far as I know. http://rockstor.com/docs/quickstart.html gives the minimum system requirements.Note the default install assumes a working DHCP ethernet network connection.
Please give details of your hardware setup regarding the network components so we can see if this is a bug or incompatible hardware.Thanks—

**Comment by phillxnet** [comment]
@Drootsam That's great thanks. OK, I think I've found what might be your problem.  Rockstor is based on CentOS 7 which is pretty similar to RHEL 7.0 and I found your motherboard mentioned in this bug report:-
```
Bug 1081362 - RHEL 7.0 unable to identify network card Realtek 8211 series - which was ok in RHEL6 [NEEDINFO]
https://bugzilla.redhat.com/show_bug.cgi?id=1081362
"While installing RHEL7.0beta to M4N68T-M LE V2 systems, the onboard ethernet controller (RTL 8211 series) wont work. But it working fine with RHEL6.0."
```
So looks like you might be affected by this, however once Rockstor has been installed it also supplants the default kernel with a much newer one that might work for you.

Work around is to use another network card, at least during install. Looks like you were just a little unlucky with this one.  You might find that once you have installed and rebooted and the new kernel is in play you may be able to return to the build in card. Nas4free and FreeNAS are both based on older FreeBSD not linux and as in the bug report this card use to work on an older RHEL version.

Hope this helps and let us know how you get on.

**Comment by Drootsam** [comment]
Thank you. I did add a new network card and that got me up and running. I then allowed the system to run the update, which took a very long time. When I went to look around the system, everything took forever, I blamed the slow Toshiba USB stick. So I put in a 1GB drive for the operating system. Now it gives me an error at the splash screen and wont continue. It still works with freenas, and Nas4free, just not with Rockstor.  Thank you for your help.
